polars/polars-time/src/chunkedarray/utf8/patterns.rs

```diff
@@ -25,26 +25,19 @@ pub(super) static DATETIME_D_M_Y: &[&str] = &[
     // --
     // supported by polars' parser
     // ---
-    // 31/12/21 12:54:48
-    "%d/%m/%y %H:%M:%S",
     // 31-12-2021 24:58:01
     "%d-%m-%Y %H:%M:%S",
     // 31-04-2021T02:45:55.555000000
     // milliseconds
     "%d-%m-%YT%H:%M:%S.%3f",
-    "%d-%m-%yT%H:%M:%S.%3f",
```

Comment:
I'm suggesting to remove the `%y` formats

First, because they typically don't work anyway, and they're shadowed by the `'%Y'` format that preceeds it
Second, because they're really ambiguous
Third, because it allows to use "at least 4 consecutive digits" as a fast-fail chec

Reply:
Sounds good. I think we should find a compromise here between the fast check and the ergonomics. And parsing two digits years sounds like silent failure indeed.